packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -87,7 +88,7 @@ export default function PostOptionsMenu({
     onBlockTags,
     onUnblockTags,
   } = useTagAndSource({
-    origin: Origin.PostContextMenu,
+    origin: AuthTriggers.PostContextMenu,
```

Comment:
I would avoid changing these actually if we can, we should opt for keeping those seperate.